task30/src/test/java/by/tut/tests/LoginTest.java

```diff
@@ -0,0 +1,50 @@
+package by.tut.tests;
+import by.tut.webpages.HomePage;
+import by.tut.webpages.LoginPage;
+import by.tut.webpages.Task2AllByElements;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import static org.testng.Assert.assertEquals;
+import java.util.concurrent.TimeUnit;
+
+public class LoginTest {
+    WebDriver driver;
+    String login = "seleniumtests@tut.by";
+    String password = "123456789zxcvbn";
+    String URL = "https://www.tut.by/";
+    LoginPage objLoginPage;
+    HomePage objHomePage;
+    Task2AllByElements objAllElements;
```

Comment:
Please add access modifier for class fields. You can use default access modifier, but here it is senseless. Also, some of them (login, password, url) are constants, so make them static final as well

Reply:
done